src/System.Runtime.Extensions/src/System/IO/TextWriter.cs

```diff
@@ -109,7 +109,9 @@ public virtual string NewLine
         // but descendant classes can override the method to provide the
         // appropriate functionality.
         //
-        public abstract void Write(char value);
+        public virtual void Write(char value)
+        {
+        }
```

Comment:
Ignore my comments... I was misunderstanding the design of the type.  It's fine as is.